<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>240</width>
				<height>189</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Explode</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_8">
							<property name="spacing">
								<number>4</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_GroupLowCount" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout">
										<property name="spacing">
											<number>10</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="label_7">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Low Groups</string>
												</property>
												<property name="alignment">
													<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_2">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="Label_GroupLowCount">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>0</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="Line" name="line">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>10</width>
											<height>25</height>
										</size>
									</property>
									<property name="frameShadow">
										<enum>QFrame::Plain</enum>
									</property>
									<property name="lineWidth">
										<number>2</number>
									</property>
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Widget_GroupHigCount" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_7">
										<property name="spacing">
											<number>10</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="Label_GroupHigCount">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>0</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_6">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>High Groups</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_02" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_2">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPlainTextEdit" name="Text_GroupLow">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPlainTextEdit" name="Text_GroupHig">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="layoutDirection">
										<enum>Qt::RightToLeft</enum>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_03" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_3">
							<property name="spacing">
								<number>12</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QDoubleSpinBox" name="Float_XMin">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>-256.000000000000000</double>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_3">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>X</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QDoubleSpinBox" name="Float_XMax">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="layoutDirection">
										<enum>Qt::RightToLeft</enum>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>256.000000000000000</double>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_04" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_4">
							<property name="spacing">
								<number>12</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QDoubleSpinBox" name="Float_YMin">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>-256.000000000000000</double>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_4">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Y</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QDoubleSpinBox" name="Float_YMax">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="layoutDirection">
										<enum>Qt::RightToLeft</enum>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>256.000000000000000</double>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_05" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_5">
							<property name="spacing">
								<number>12</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QDoubleSpinBox" name="Float_ZMin">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>-256.000000000000000</double>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_5">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Z</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QDoubleSpinBox" name="Float_ZMax">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="layoutDirection">
										<enum>Qt::RightToLeft</enum>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="buttonSymbols">
										<enum>QAbstractSpinBox::PlusMinus</enum>
									</property>
									<property name="decimals">
										<number>3</number>
									</property>
									<property name="minimum">
										<double>-999999.000000000000000</double>
									</property>
									<property name="maximum">
										<double>999999.000000000000000</double>
									</property>
									<property name="value">
										<double>256.000000000000000</double>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_6">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_Apply">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Apply</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Revert">
									<property name="enabled">
										<bool>false</bool>
									</property>
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Revert</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>